backend/src/main/java/com/google/rolecall/services/SectionServices.java

```diff
@@ -340,6 +305,29 @@ public void deleteSection(int id) throws EntityNotFoundException, InvalidParamet
     sectionRepo.deleteById(id);
   }
 
+  // Utility functions
+
+  private void UpdateRevelationChildren(Section section, Integer[] siblingIndexArray,
```

Comment:
Thanks for extracting this! Let's make sure it's camelCase `updateRevelationChildren`

Reply:
Done.